examples/js-cdn-typescript/README.md

```diff
@@ -0,0 +1,43 @@
+# Web app with Javascript CDN
+
+This is a basic web application written in TypeScript that uses RudderTyper client library along with JS cdn script.
+
+## Prerequisite(If you want to use your tracking plan)
+
+1. You have setup the tracking plan from the dashboard.
+2. Generated PAT from dashboard settings.
+3. Did the authentication step using PAT and email to generate `ruddertyper.yml` and `plan.json` file in the root level of the repo.
+4. Move ruddertyper.yml file from root level to examples/js-cdn-typescript this folder.
+5. Replace the plan.json in js-cdn-typescript/src/analytics with the one you generated.
+6. Delete `index.ts` and `rudder.ts` inside analytics folder.
+7. Run the command `npx rudder-typer`
+8. New `index.ts` and `rudder.ts` file will be generated.
+
+## Steps to run the sample app
+
+1. From terminal go to project folder.
+2. Run the command `npm run setup`
+3. Run the command `browserify src/analytics/index.ts -p [ tsify ] --standalone rudderTyper > public/rudderTyperBundle.js`
```

Comment:
`browserify` should be added as a dev dependency.

Also, how does this output compare to the JavaScript output from RudderTyper?

Reply:
This is a single file with all the dependencies. Kind of the minified file we generate for JS SDK.